cross/urw-base35-fonts/Makefile

```diff
@@ -0,0 +1,28 @@
+PKG_NAME = urw-base35-fonts
+PKG_VERS = 20151005
+PKG_EXT = zip
+PKG_DIST_NAME = urw-base35-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://ftp.icm.edu.pl/pub/unix/postscript/ghostscript/fonts
+# download and extract only
+EXTRACT_PATH = $(WORK_DIR)/share/fonts
+
+DEPENDS =
+
+# arch independent
+override ARCH=
+
+HOMEPAGE = https://www.ghostscript.com/doc/current/Fonts.htm
+COMMENT  = Core Font Set containing 35 freely distributable fonts from (URW)++
+LICENSE  = GNU AGPLv3
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = nop
+INSTALL_TARGET = urw-base35-fonts
+
+
+include ../../mk/spksrc.cross-cc.mk

```

Comment:
We may find a better way to include such "resources", knowing that `cross/` and `cross-cc` means "cross c-compilation".